docs/index.md

```diff
@@ -138,14 +141,16 @@ with fsspec.open('file://my/file/path', 'rb') as fhandle:
 filebytes[:256]
 ```
 
-.. note::
-In disctrubuted environments, (for e.g. remote kernels) the paths in the code
+:::note
+**Note:** In disctrubuted environments, (for e.g. remote kernels) the paths in the code
 that the helper uses may not be valid unless the kernel and server share a filesystem.
+:::
 
-.. warning::
-The environment variable "JUPYTER_FSSPEC_ALLOW_ABSOLUTE_PATHS" defaults to true, and
```

Comment:
I have not come across `:::`. It is an alternative to indenting the block following the `::` we had before?

Reply:
The `.. warning::` block was not rendering, I believe because it is `.rst` syntax while this update uses the [colon-fence myst extension](https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#code-fences-using-colons). I think another option is to switch to `.rst`?

Reply:
You are right, this is rst syntax - I didn't realise that the file wasn't! It's good to know you can do the same in .md (although it doesn't render if you view the file in the github UI).